frontend/lib/src/components/shared/Icon/styled-components.ts

```diff
@@ -82,6 +82,32 @@ export const StyledIcon = styled("span", {
   }
 })
 
+export interface StyledDynamicIconProps {
+  iconValue: string
```

Comment:
I think iconValue and testid can also be removed here since it's not used by the styled component